lib/ability.rb

```diff
@@ -56,8 +56,12 @@ def initialize(user, resource, controller = nil)
     end
 
     if user.billing_administrator?
-      can :manage, [Account, Journal, Order, OrderDetail, Reservation]
-      cannot :administer, [Order, OrderDetail, Reservation]
+      can :manage, [Account, Journal, OrderDetail]
+      can [:send_receipt, :show], Order
+      if resource == Facility.cross_facility
+        can :manage, [Reservation, User]
```

Comment:
Would it make more sense to make this can be a whitelist of only what they _can_ do rather than `:manage` and remove abilities?